### ISSUE TITLE
Questions : Know Topic Response

### ISSUE THREAD

**Issue by nehasharma1997** [kreait/laravel-firebase#12]
Hi, This is a question, I do not know where to ask, if this is not a good place, then sorry for it. I used `FirebaseMessaging::send($message)` to send message to a topic and it return `"name": "projects/projectname/messages/2555307708117287325"`
Now can I know how many devices to message deliver ? Is message deliver successfully etc ?

**Comment by jeromegamez** [comment]
To be honest, at the moment I don't know what the returned value can be used for - when sending a message to be processed to the Firebase Servers, this is what they return, and I return it in the SDK just in case it can be used for _something_ :).

Besides that, there is just limited insight that you can gain after sending a message:

When you send it and the SDK doesn't throw an Error, the message has been successfully delivered to the Firebase Servers. At this point, the Firebase Servers just responded "The message is valid, we have received it, and we'll take it from here"

For further insight, have a look at https://firebase.google.com/docs/cloud-messaging/understand-delivery.

The gist is:

- You can see some general statistics in the Cloud Messaging Reporting Dashboard in your Project's Firebase console, but only how many have been sent, received, viewed and opened - and this only reliably on Android devices, partially on iOS devices and probably not at all in web applications.
- You can enable Android apps to send Notification metrics to Google BigQuery (this seems not possible for iOS or Web apps, at least at the moment)
- When you compose messages to be sent, you can add FCM Analytics Labels that you can filter by, but I can't tell you how well that works 😅 

In summary: If you don't get an error, it worked, but we don't know how well it worked :)

**Comment by nehasharma1997** [comment]
Thank you for your reply.
